docs/TroubleshootingTips.md

```diff
@@ -56,8 +74,8 @@ Fast dev way
 [Chrome Dev tool Clear Cache & Cookies + Hard Reload](https://developers.google.com/web/updates/2015/05/hard-reload).
 Press F12 (opens dev window) then right click on the refresh arrow and hit Empty Cache and Hard Reload 
 
-#### Factory reset 
+### Factory reset 
 
-If you know you are having a browser issue with Enklu and can't find a way to fix it a factory reset may be best.  If you use a password managers make sure you remember your master password first. A factory reset will clear all browser settings. This also will turn off any extensions.  (This won't remove extensions but clears their settings, disables them, and resets login-in info).  It will also clear permissions and how content is loaded.
+If you know you are having a browser issue with Enklu and can't find a way to fix, it a factory reset may be best. If you use a password manager, make sure you remember your master password first. A factory reset will clear all browser settings, as well as turn off any extensions. (This won't remove extensions but clears their settings, disables them, and resets login-in info). It will also clear permissions and how content is loaded.
```

Comment:
oh, should probably move the comma after "it": "and can't find a way to fix it, a factory reset.."